src/renderer/settings/values/networks.js

```diff
@@ -1,8 +1,9 @@
 import { settings } from '@cityofzion/neon-js';
 import { keys } from 'lodash';
 
-export const NOS_LOCAL = 'nOSLocal';
-export const NOS_TESTNET = 'nOSNet';
+export const DEFAULT_NET = 'nOSNet';
 
+export const NOS_TESTNET = 'nOSNet';
+export const NOS_LOCAL = 'nOSLocal';
```

Comment:
Just so we're not redefining the same string twice, can we change this to:

```js
export const NOS_TESTNET = 'nOSNet';
export const NOS_LOCAL = 'nOSLocal';

export const DEFAULT_NET = NOS_TESTNET;
```

Reply:
I was thinking the same when I wrote it 😂 Didn't persist on doing it hehe